packages/editor/src/components/provider/index.js

```diff
@@ -208,8 +207,11 @@ class EditorProvider extends Component {
 				'titlePlaceholder',
 			] ),
 			mediaUpload: hasUploadPermissions ? mediaUpload : undefined,
-			__experimentalReusableBlocks: reusableBlocks,
-			__experimentalFetchReusableBlocks,
+			__experimentalReusableBlocks: reusableBlocks?.map( ( c ) => ( {
+				...c,
+				content: c.content.raw,
+				title: c.title.raw,
+			} ) ),
```

Comment:
Noting that the code downstream would ideally expect the content and title in this format.